docs/contributing/resoto.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: Resoto Components
+---
+
+# Contributing to Resoto Components
+
+The source code for Resoto lives in the [`someengineering/resoto` repository on GitHub](https://github.com/someengineering/resoto).
+Resoto consists of multiple components, where each component is maintained as separate project.
+A change should target one of the components.
+
+- [Resoto Core](https://github.com/someengineering/resoto/tree/main/resotocore)
+- [Resoto Shell](https://github.com/someengineering/resoto/tree/main/resotoshell)
+- [Resoto Worker](https://github.com/someengineering/resoto/tree/main/resotoworker)
+- [Resoto Metrics](https://github.com/someengineering/resoto/tree/main/resotometrics)
+- [Resoto Library](https://github.com/someengineering/resoto/tree/main/resotolib)
+
+## Authoring Changes
+
+Contributions are made via [pull requests to the GitHub repository](https://github.com/someengineering/resoto/pulls).
+You will first need to [fork](https://docs.github.com/get-started/quickstart/fork-a-repo) the repository.
+
+#### Prerequisites
+
+- [Git](https://git-scm.com)
+- [Python](https://www.python.org) Version 3.8 or later is required. We recommend 3.10.
+- [ArangoDB](https://www.arangodb.com) Version 3.8.2 or later is required.
+- Optional [gcc](https://gcc.gnu.org) Depending on the host system, dependant python packages need to be compiled from source.
+
+#### Set Up a Virtual Environment
+
+We recommend using a Python [virtual environment](https://docs.python.org/3/tutorial/venv.html) to develop.
+A script is part of the Resoto code base to simplify this process and will prepare your virtual environment.
+
+```bash
+./setup_venv.sh --dev --path .
+```
+
+After the virtual env has been created, it needs to be activated.
+Linux and macOS:
+
+```bash
+source venv/bin/activate
+```
+
+or Windows:
+
+```powershell
+venv\Scripts\activate.bat
+```
+
+#### Start the database
+
+After you have installed ArangoDB, you need to start it.
```

Comment:
Was the DB created by the script above? If not, need to go into more detail here.

Reply:
Yes - just starting the DB is sufficient.

Reply:
How would the user start the DB, in that case?

Reply:
added a sentence to that.